hack/image-cross.sh

```diff
@@ -46,6 +56,7 @@ for ARCH in "${ARCHES[@]}"; do
         .
     for T in "${TAGS[@]}"; do
         docker push "$IMAGE-$ARCH:$T"
+        cosign sign "$IMAGE-$ARCH:$T"
```

Comment:
Will we use cosign's keyless signing? I think you need to provide a reference to the OIDC issuer as a command-line parameter.